v2/api/appconfiguration/v1api20220501/configuration_store_types_gen.go

```diff
@@ -173,95 +173,107 @@ func (store *ConfigurationStore) SetStatus(status genruntime.ConvertibleStatus)
 var _ admission.Validator = &ConfigurationStore{}
 
 // ValidateCreate validates the creation of the resource
-func (store *ConfigurationStore) ValidateCreate() error {
+func (store *ConfigurationStore) ValidateCreate() (admission.Warnings, error) {
 	validations := store.createValidations()
 	var temp any = store
 	if runtimeValidator, ok := temp.(genruntime.Validator); ok {
 		validations = append(validations, runtimeValidator.CreateValidations()...)
 	}
 	var errs []error
+	var warnings admission.Warnings
 	for _, validation := range validations {
-		err := validation()
+		warning, err := validation()
+		if warning != nil {
+			warnings = append(warnings, warning...)
+		}
 		if err != nil {
 			errs = append(errs, err)
 		}
 	}
-	return kerrors.NewAggregate(errs)
+	return warnings, kerrors.NewAggregate(errs)
```

Comment:
Can you use your new helper method in `genruntime` to do this?

Reply:
That's a good idea, that'll reduce a lot of generated code as well

Reply:
Done!